src/react-guides/consts.ts

```diff
@@ -87,11 +87,11 @@ canvas {
 }
 
 .light {
-    background: linear-gradient(90deg, rgba(255,255,255,0) 0%, rgba(255,255,255,0.6) 15%, rgba(255,255,255,1) 35%, rgba(255,255,255,1) 50%, rgba(255,255,255,1) 65%, rgba(255,255,255,0.6) 85%, rgba(255,255,255,0) 100%);
+    background: linear-gradient(90deg, rgba(255,255,255,0) 0%, rgba(255,255,255,0.4) 15%, rgba(255,255,255,0.4) 35%, rgba(255,255,255,0.4) 50%, rgba(255,255,255,0.4) 65%, rgba(255,255,255,0.4) 85%, rgba(255,255,255,0) 100%);
 }
 
 .dark {
-    background: linear-gradient(90deg, rgba(68,68,64,0) 0%, rgba(68,68,68,0.6) 15%, rgba(68,68,68,1) 35%, rgba(68,68,68,1) 50%, rgba(68,68,68,1) 65%, rgba(68,68,68,0.6) 85%, rgba(68,68,68,0) 100%);
+    background: linear-gradient(90deg, rgba(68,68,64,0) 0%, rgba(68,68,68,0.6) 15%, rgba(68,68,68,0.9) 35%, rgba(68,68,68,0.9) 50%, rgba(68,68,68,0.9) 65%, rgba(68,68,68,0.6) 85%, rgba(68,68,68,0) 100%);
```

Comment:
I think it exceed max line length

Reply:
@nickbohdanov  Yes, thank you, I fixed it